odl/solvers/advanced/proximal_operators.py

```diff
@@ -31,7 +31,7 @@
 from builtins import super
 
 import numpy as np
-import scipy.special
+import scipy
```

Comment:
I think I used `scipy.special.lambertw`further down anyway. But now that the build fails I will have to double check...